packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -59,6 +59,9 @@ class ExportToCsvCommand extends Command {
       multiple: false,
       required: false,
     }),
+    "team-uid": flags.string({
+      description: 'Uid of the team whose user data and stack roles are required'
+    })
```

Comment:
Have we reviewed this text with doc team.?

Reply:
Not yet, I am preparing the sheet